GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifyAccountDetail.h

```diff
@@ -43,6 +43,9 @@ typedef void (^GIDVerifyCompletion)(GIDVerifiedAccountDetailResult *_Nullable ve
 /// The active configuration for this instance of `GIDVerifyAccountDetail`.
 @property(nonatomic, nullable) GIDConfiguration *configuration;
 
+- (instancetype)initWithConfig:(nullable GIDConfiguration *)config
```

Comment:
Also, perhaps I'm forgetting, but should `config` be `nullable`? Isn't the purpose of using this initializer to pass in a `GIDConfiguration`? If passing nil, why not just use the parameter-less initializer? Maybe I'm missing a use-case...

Reply:
I believe this was done following [this comment](https://github.com/google/GoogleSignIn-iOS/pull/401#discussion_r1585529200) earlier in the PR but maybe I misunderstood. I do prefer to keep it as non-nullable as well.

Reply:
Ohhh, I think it's because of the parameter-less init:

```
- (instancetype)init {
  GIDConfiguration *configuration;
  NSBundle *bundle = NSBundle.mainBundle;
  if (bundle) {
    configuration = [GIDConfiguration configurationFromBundle:bundle];
  }

  return [self initWithConfig:configuration];
}
```

The above calls `+[GIDConfiguration configurationFromBundle:]`, which [can return nil](https://github.com/google/GoogleSignIn-iOS/blob/briannamorales/vwg-flow/GoogleSignIn/Sources/GIDConfiguration.m#L78). Thus, `[self initWithConfig:configuration]` can pass in a nil `GIDConfiguration`. 

I think `-[GIDVerifyAccountDetail initWithConfig:]` should _not_ be marked `nullable`. If the configuration is being passed in, then why pass in nil?

Furthermore, the parameter-less `init` should probably be annotated as `nullable` and return nil if the automatically created `GIDConfiguration` is nil. We need the configuration.

Reply:
Ahh makes sense, added!